content/docs/contributing/release-process.md

```diff
@@ -141,15 +143,160 @@ page if a step is missing or if it is outdated.
+   [^1]: One or more "patch pre-releases" may be created to allow voluntary community testing of a bug fix or security fix before the fix is made generally available. The suffix `-beta` must be used for patch pre-releases.
 
-2. **(final release only)** Make sure that a PR with the new upgrade
+2. Set the 4 environment variables by copying the following snippet in your
+   shell table:
+
+     ```bash
+     export RELEASE_VERSION="v1.3.0-alpha.0"
+     export START_TAG="v1.2.0"
+     export END_REV="release-1.3"
+     export BRANCH="release-1.3"
+     ```
+
+    > **Note:** To help you fill in the correct values, use the following
+    > examples:
+    >
+    > | Variable          | Example 1        | Example 2        | Example 2        | Example 3     | Example 4     |
+    > | ----------------- | ---------------- | ---------------- | ---------------- | ------------- | ------------- |
+    > |                   | initial alpha    | subsequent alpha | beta release     | final release | patch release |
+    > | `RELEASE_VERSION` | `v1.3.0-alpha.0` | `v1.3.0-alpha.1` | `v1.3.0-beta.0`  | `v1.3.0`      | `v1.3.1`      |
+    > | `START_TAG`       | `v1.2.0`         | `v1.3.0-alpha.0` | `v1.3.0-alpha.1` | `v1.2.0`\*    | `v1.3.0`      |
+    > | `END_REV`         | `master`         | `master`         | `release-1.3`    | `release-1.3` | `release-1.3` |
+    > | `BRANCH`          | `master`         | `master`         | `release-1.3`    | `release-1.3` | `release-1.3` |
+    >
+    > \*Do not use a patch here (e.g., no `v1.2.3`). It must be `v1.2.0`:
+    > you must use the latest tag that belongs to the release branch you are
+    > releasing on; in the above example, the release branch is
+    > `release-1.3`, and the latest tag on that branch is `v1.2.0`.
+
+    > **Note:** The 4 variables are described in [the README of the
+   `release-notes`
+   tool](https://github.com/kubernetes/release/blob/master/cmd/release-notes/README.md#options).
+   For your convenience, the following table summarizes what you need to know:
+    >
+    > | Variable          | Description                             |
+    > | ----------------- | --------------------------------------- |
+    > | `RELEASE_VERSION` | The git tag                             |
+    > | `START_TAG`\*     | The git tag of the "previous"\* release |
+    > | `END_REV`         | Name of your release branch (inclusive) |
+    > | `BRANCH`          | Name of your release branch             |
+
+3. **(final release only)** Prepare the Website "Upgrade Notes" PR.
+
+   Make sure that a PR with the new upgrade
    document is ready to be merged on
    [cert-manager/website](https://github.com/cert-manager/website). See for
    example, see
    [upgrading-1.0-1.1](https://cert-manager.io/docs/installation/upgrading/upgrading-1.0-1.1/).
 
-3. Check that the `origin` remote is correct. To do that, run the following
+4. **(final + patch releases)** Prepare the Website "Release Notes" PR.
```

Comment:
4 and 5 have the same title "Prepare the Website "Release Notes" PR."

Also, half of the release process is just about generating the release notes, and that seems to be very complex for some reason. Would it be possible to simplify this process & maybe move it into a seperate section on this website?

Reply:
Yes. That said, i would prefer having the release process in a good shape right now so that anyone can release. Simplifying the process would take a lot more time than just documenting it, that’s why I prefer focusing on documenting it well regardless of the large number of steps

It is still possible to move stuff around later

Reply:
Done, I have fixed the section headings